docs/package.json

```diff
@@ -22,7 +22,6 @@
     "@storybook/addon-essentials": "^6.5.12",
     "@storybook/addon-interactions": "^6.5.12",
     "@storybook/addon-links": "^6.5.12",
-    "@storybook/addon-storyshots": "^6.5.15",
```

Comment:
looks like removal of this is critical to resolve the storybook site not running properly